src/compile/unit.ts

```diff
@@ -65,7 +65,6 @@ export class UnitModel extends Model {
     // TODO?: ideally we should use config only inside this constructor
     const config = this._config = this._initConfig(spec.config, parent);
 
-    // FIXME move stacked out of config as it's not really a theme.
     // calculate stack properties
     this._stack = stack(mark, encoding, config.mark.stacked);
```

Comment:
@domoritz What bugs me a bit is internally stack is a Spec level operation, not an encoding level operation.